charts/pelorus/charts/exporters/templates/_buildconfig.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   source:
     contextDir: {{ .source_context_dir }}
     git:
-      ref: {{ .source_ref }}
+      ref: {{ .source_ref | default "stable" }}
```

Comment:
I fear that this change, paired with removing source_ref from variables fingers will result in different behavior than a user will be expecting, especially if they have cloned the master branch and could also contribute to a feeling of "black magic". I think defaulting to stable makes more sense once we have an install path that doesn't involve cloning a repo (e.g. using images and a published helm chart), or when installing a release. Thoughts?